nessie/sanity_api_test.go

```diff
@@ -0,0 +1,167 @@
+package nessie
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/go-openapi/swag"
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/api/gen/client/branches"
+	"github.com/treeverse/lakefs/api/gen/client/commits"
+	"github.com/treeverse/lakefs/api/gen/client/objects"
+	"github.com/treeverse/lakefs/api/gen/client/refs"
+	"github.com/treeverse/lakefs/api/gen/client/repositories"
+	"github.com/treeverse/lakefs/api/gen/models"
+)
+
+func TestSanityAPI(t *testing.T) {
```

Comment:
 Sanity is too broad IMO. Maybe `TestDiffBranches` as diff is the action which isn't covered by other tests?

Reply:
wanted a simple flow that include all the basic functionality - the next PR will include the same steps using the AWS/gateway interface.
If you have a better name than Sanity? BasicFlow? SimpleUseCase? I'll take it.
Testing diff and merge can be covered by all the options in separate system test file.

Reply:
If it's not specific then any name will do.